config/config.go

```diff
@@ -42,7 +45,18 @@ func Get() (*Config, error) {
 		ResultProducerTopic:      "import-observations-inserted",
 		CacheTTL:                 time.Minute * 60,
 		GracefulShutdownTimeout:  time.Second * 10,
+		ServiceAuthToken:         "AA78C45F-DD64-4631-BED9-FEAE29200620",
+		ZebedeeURL:               "http://localhost:8082",
 	}
 
+	cfg.ServiceAuthToken = "Bearer " + cfg.ServiceAuthToken
+
 	return cfg, envconfig.Process("", cfg)
 }
+
+// String is implemented to prevent sensitive fields being logged.
+// The config is returned as JSON with sensitive fields omitted.
+func (config Config) String() string {
```

Comment:
This function adds no value, the log package will remove any json:"-" values anyway